docs/tutorials/datasets/mhp_v1.py

```diff
@@ -0,0 +1,92 @@
+"""Prepare Multi-Human Parsing V1 dataset.
+========================
+
+`Multi-Human Parsing V1 (MHP-v1) <https://github.com/ZhaoJ9014/Multi-Human-Parsing/>`_ is a human-centric containing
+5 thousands images annotated with 18 object categories.
+This tutorial help you to download MHP-v1 and set it up for later experiments.
+
+.. image:: https://github.com/ZhaoJ9014/Multi-Human-Parsing_MHP/blob/master/Figures/Fig1.png
+   :width: 600 px
+
+.. hint::
+
+   You need 850 MB free disk space to download and extract this dataset.
+   SSD harddrives are recommended for faster speed.
+   The time it takes to prepare the dataset depends on your Internet connection
+   and disk speed. For example, it takes around 1 mins on an AWS EC2 instance
```

Comment:
Just want to confirm, 1 min for everything? Please double check.

Reply:
Yes, it just spend about 1 mins for downloading and extracting the datasets on Amazon EC2 p3.16xlarge.

Reply:
Good to know, thanks.